Vic2ToHoI4/Source/HOI4World/Sounds/SoundEffectsMapper.cpp

```diff
@@ -0,0 +1,15 @@
+#include "SoundEffectsMapper.h"
+
+
+
+std::set<HoI4::SoundEffectsMapping> HoI4::SoundEffectsMapper::getMappings(const std::string& culture)
```

Comment:
why not optional?

Reply:
An empty set indicates no sound effects just fine.

Reply:
I had a brainfart, sorry. Of course being empty is fine..